tests/js/run_docker.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
```

Comment:
This file runs the same commands we had in `package.json`, only in a more readable format and allowing us to change the `theme` dynamically.